src/js/helpers.js

```diff
@@ -237,8 +237,8 @@ export default class Helpers {
                 }
               } else if (fieldData.subtype === 'tinymce' && window.tinymce) {
                 const id = `${fieldData.name}-preview`
-                if (window.tinymce.editors[id]) {
-                  const editor = window.tinymce.editors[id]
+                const editor = window.tinymce.get(id)
```

Comment:
better syntax!

Reply:
Also window.tinymce.editors is removed in later versions